Factory/FactoryMethod/AccountFactory.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Factory.FactoryMethod
+{
+    public static class AccountFactory
```

Comment:
Why `static` and not inherited from the interface?

Reply:
I saw it can be used as static class and method, but np I'll change everything